### ISSUE TITLE
Difficulty seeing with the blue background in 'Dark Mode'

### ISSUE THREAD

**Issue by zededd** [klembot/twinejs#1240]
### Is your feature request related to a problem? Please describe.

I'm colourblind and blue on blue is just screwing with what I can see.
I would rather roll back than use Twine as it is now!

### Describe the solution you'd like.

An option to change this colour, or remove it and replace it with something neutral

### Describe alternatives you've considered.

I suggested this feature back when Twine 2 was released and for a while it was not needed as the background was not as difficult to deal with

### Presubmission checklist

- [ ] I am interested in working on code that would implement this feature request. (This is not required to submit a suggestion.)
- [X] I have done a search and believe that an issue does not already exist for this idea in the GitHub repository.
- [X] I have read and agree to abide by this project's Code of Conduct.

**Comment by AlyxMS** [comment]
You can go to Twine - Perferences and change the theme to Light.

**Comment by webbedspace** [comment]
>You can go to Twine - Perferences and change the theme to Light.

IMHO this is not a solution at all - Both themes should be equally usable by everyone.

**Comment by klembot** [comment]
Could you attach a screenshot of the blue-on-blue you're having trouble with? I'm guessing it's a selected passage  but I want to be sure.

Also, could you be more specific as to what the difficulty is? e.g. is it distinguishing selected passages from unselected ones? Distinguishing selected passages from the background? Reading the text? The reason I ask is that in order to find a suitable solution, I need to understand the problem better.

**Comment by zededd** [comment]
I am having difficulty with the background. The space between passages that should be easy to obtain examples (I am not updating again for the sake of one screenshot, only to have to uninstall to 'downgrade' again. my apologies).

Please do not assume this has anything to do with the objects, nor anything else than the blue-on-blue background.
Due to the change in colour pallete between versions, it is just impossible to see.
I confess that I did not try the 'light theme' as having a dark pallete over the other programs I am using (many of them customisable to some degree) this would make Twine very bright by comparison and even harder to see by contrast, even though the problem would not be with Twine's palette in that case.

I can appreciate that this is not a common issue, but the option to change the background to anything else would be greatly appreciated. Grey on black has proven to be not only common, but prefered across multiple platforms, operating systems and websites.

**Comment by hituro** [comment]
Example of Light theme for the benefit of @zededd 
![image](https://user-images.githubusercontent.com/4206142/185800874-e898c6fd-fb6b-47a3-af9f-a5b3e50a8a81.png)

**Comment by TurtleXing** [comment]
I am also having this issue. I am also colorblind and my issue stems from the saturated blue of the background jumping forward while the unsaturated grey of the passages falls back. It makes it hard to distinguish the edges of passages and confusing to look at, like I am trying to work on top of a optical illusion. 


I have a semi-annoying work around of opening up the debug console and editing the 'faint-blue' color to something way less saturated and darker. However this is a change that needs to be made every time twine is reopened and I haven't been able to figure out how to set this to a default value for Dark Theme.
![Capture](https://user-images.githubusercontent.com/6423239/202060993-e345f1c3-4689-48d0-82bb-4dad00f8aa5d.PNG)

**Comment by klembot** [comment]
Something I'd like to do in a future release would be to add high contrast versions of the light and dark themes, which I think could address this problem.

**Comment by david-donachie** [comment]
> Something I'd like to do in a future release would be to add high contrast versions of the light and dark themes, which I think could address this problem.

Certainly my urge to replace it might have been lessened if it was closer to the colour palette of the game, but I might still have wanted to replace the spinner with a rotating flower wreath, so I don't know ...

**Comment by NickHeathfield** [comment]
Apart from issues with colour-blindness, using a rich deep blue for a dark mode whose whole point is to be easy on the eye, is a terrible idea, especially with the grey-on-slightly-darker-grey text boxes that make up the actual content, which is just as terrible.

Suggest reverting the colour scheme to how is was in 2.3 or something. That was at least bearable.

**Comment by TurtleXing** [comment]
No update on this? 8 Month old issue and I still can't update past V2.3.16 (released 14 months ago) due to a simple accessibility issue.

**Comment by NickHeathfield** [comment]
IKR? I don't have the time or knowledge to do a lot of iterative compiling and testing for this, but for heaven's sake, someone who does, just go over this quick hack photoshopping with a colour picker and plug the values into twinejs/src/styls/colors.css

Improved the contrast and shifted the colour to green because eye strain is a bigger concern than branding.

![image](https://user-images.githubusercontent.com/17346217/230578935-e4ab9ce6-d1f8-4e42-8b85-de590e1e4f88.png)

![image](https://user-images.githubusercontent.com/17346217/230578999-2f85498b-a3dc-484b-a4e4-936ed9bc9c75.png)

**Comment by TurtleXing** [comment]
> IKR? I don't have the time or knowledge to do a lot of iterative compiling and testing for this, but for heaven's sake, someone who does, just go over this quick hack photoshopping with a colour picker and plug the values into twinejs/src/styls/colors.css
> 
> ![image](https://user-images.githubusercontent.com/17346217/230578935-e4ab9ce6-d1f8-4e42-8b85-de590e1e4f88.png)
> 
> ![image](https://user-images.githubusercontent.com/17346217/230578999-2f85498b-a3dc-484b-a4e4-936ed9bc9c75.png)
> 
> Improved the contrast and shifted the colour to green because eye strain is a bigger concern than branding. A box shadow or border around each item would be nice too, to better separate it from the background. Yarn Editor does this to good effect:
> 
> ![image](https://user-images.githubusercontent.com/17346217/230579821-51e8b4c2-4bc3-48ab-951d-18482abafeed.png)

Yeah. I asked the discord forever ago to see if anyone had solutions for changing the default colour values that persist after closing the app but it never amounted to anything. I spent hours opening the files and changing random colour values to see if I could find the right one but nothing ever stuck there either.

**Comment by TurtleXing** [comment]
Almost two years later I found a solution. 

Download 7-Zip https://www.7-zip.org/ and the Asar7z Plugin http://www.tc4shell.com/en/7zip/asar/
Open Twine\Resources\app.asar with 7zip
Navigate to electron-build\renderer\static\css\ and extract main.2d3369b7.css
Open the extracted file with any text editor and search for 'faint-blue' (faint blue is the colour used for the background. Changing this probably changes other things that use that faint blue but whatever, it makes it usable) change the hex code to whatever hex code you like, #000000 is black
Save and drag the file back into the 7zip where you found it, replacing the old file.
Reopen Twine and enjoy a usable UI.

**Comment by klembot** [comment]
https://github.com/klembot/twinejs/pull/1415 adds the ability to customize Twine's UI without having to unpack it.
